modules/test/scripts/init-couchbase/init-buckets.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# expand variables and print commands
+set -o xtrace
+
+# exit immediately if a command fails or if there are unset vars
+set -euo pipefail
+
+CB_USER="${CB_USER:-Administrator}"
+CB_PSWD="${CB_PSWD:-password}"
+
+CB_BUCKET_RAMSIZE="${CB_BUCKET_RAMSIZE:-128}"
+
+echo "couchbase-cli bucket-create travel-sample..."
+/opt/couchbase/bin/couchbase-cli bucket-create \
+-c localhost -u ${CB_USER} -p ${CB_PSWD} \
```

Comment:
Missing indentation on these continuation lines looks odd.

Reply:
(and below throughout. not sure what our coding standard is here, but I've always seen them indented.)

Reply:
I believe it's my `bash-beautify` VSCode plugin, it formats on save (https://marketplace.visualstudio.com/items?itemName=shakram02.bash-beautify)

Reply:
It's been handy, but in this particular case it just seems to remove the indent, I'm not really sure what the standard is here myself. Let me switch it off and add an indent.

Reply:
I guess this is also following some kind of prettifier standard, so I might just leave it as is